modules/storages/app/models/storages/storage.rb

```diff
@@ -119,6 +119,19 @@ def self.extract_part_from_piped_string(text, index)
       end
     end
 
+    def oauth_access_granted?(user)
```

Comment:
:yellow_circle: I have the feeling that we should name this method differently now that it also includes SSO... But I can't come up with a good proposal myself, so I guess I am leaving this thought here, but consider it very optional to do anything about it.

I am thinking about something along the lines of `user_connected?` maybe, but not really satisfied with it.  
Thinking about that naming proposal it's also interesting that sometimes we check the presence of a remote identity (e.g. `authorizationState`) and in other cases (here) we check the presence of certain tokens to decide whether a user is hooked up to a storage.

**tl;dr:** Nothing really to do here, unless you take my feedback as inspiration for something :see_no_evil:

Reply:
I was thinking about the better name, but did not come up with something satisfying either...